pymove/models/pattern_mining/clustering.py

```diff
@@ -44,7 +46,8 @@ def elbow_method(move_data, k_initial=1, max_clusters=15, k_iteration=1):
 
 
 def gap_statistic(move_data, nrefs=3, k_initial=1, max_clusters=15, k_iteration=1):
-    """Calculates optimal clusters numbers using Gap Statistic from Tibshirani, Walther, Hastie
+    """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5419895)